Chapter3/Chapter3.Taask5/Program.cs

```diff
@@ -7,9 +7,12 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
-            for (int a = 1; a < 10; a++ )
```

Comment:
1. Task description says that you have to read out the number from console(NOT HARDCODE it)
2. you have to do it with while and do-while loops. Foreach should be skipped, it is not possible.